web/src/index.js

```diff
@@ -3,8 +3,18 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './service-worker';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import {ApolloProvider} from 'react-apollo'
+import ApolloClient from 'apollo-boost'
 
-ReactDOM.render(<App/>, document.getElementById('root'));
+const client = new ApolloClient({
+    uri: "http://localhost:15780"
```

Comment:
How will this change after we deploy? Set up different scenarios (dev, prod, test) to handle the client addresses. OK to leave a TODO and punt till next PR.